src/main/java/org/jboss/qa/jcontainer/Container.java

```diff
@@ -60,27 +58,15 @@ public synchronized void start() throws Exception {
 		}
 		final List<String> cmd = configuration.generateCommand();
 		cmd.addAll(configuration.getParams());
-		logger.debug("Process arguments: " + cmd.toString());
+		log.debug("Process arguments: " + cmd.toString());
 
 		final ProcessBuilder processBuilder = new ProcessBuilder(cmd);
-
-		// Adjust JAVA_OPTS
-		final StringBuilder javaOpts = new StringBuilder();
-		if (configuration.getXms() != null) {
-			javaOpts.append(" -Xms" + configuration.getXms());
-		}
-		if (configuration.getXmx() != null) {
-			javaOpts.append(" -Xmx" + configuration.getXmx());
-		}
-		if (configuration.getMaxPermSize() != null) {
-			javaOpts.append(" -XX:MaxPermSize=" + configuration.getMaxPermSize());
-		}
-		javaOpts.append(" " + configuration.getEnvProps().get("JAVA_OPTS"));
-		javaOpts.append(" " + System.getenv("JAVA_OPTS"));
-		//configuration.getEnvProps().put("JAVA_OPTS", javaOpts.toString()); // TODO(mbasovni): fix JAVA_OPTS
-
+		processBuilder.environment().putAll(System.getenv());
 		processBuilder.environment().putAll(configuration.getEnvProps());
 
+//		processBuilder.redirectOutput(ProcessBuilder.Redirect.INHERIT);
+//		processBuilder.redirectError(ProcessBuilder.Redirect.INHERIT);
```

Comment:
If you uncomment those lines, server standard output will be merged with current standard output. I used it just for debugging. I can delete it.